src/www/views/servers_view/server_list_item/server_hero_card.ts

```diff
@@ -0,0 +1,232 @@
+/*
+  Copyright 2021 The Outline Authors
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import '@material/mwc-button';
+import '@material/mwc-icon-button';
+import '@material/mwc-menu';
+import '../server_connection_indicator';
+
+import {html, css} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {createRef, Ref, ref} from 'lit/directives/ref.js';
+import {Menu} from '@material/mwc-menu';
+
+import {ServerConnectionState} from '../server_connection_indicator';
+import {ServerListItemElement} from '.';
+
+@customElement('server-hero-card')
+export class ServerHeroCard extends ServerListItemElement {
+  @property({attribute: 'root-path'}) rootPath: string;
+  @property() localize: (messageID: string) => string;
+
+  menuRef: Ref<Menu> = createRef();
+
+  static styles = css`
+    /* TODO(daniellacosse): global css reset */
+    h2,
+    span,
+    footer,
+    button,
+    div {
+      all: initial;
+    }
+
+    :host {
+      --server-name-size: 1rem;
+      --server-address-size: 0.875rem;
+
+      align-items: center;
+      background: var(--outline-card-background);
+      border-radius: var(--outline-corner);
+      box-shadow: var(--outline-elevation);
+      display: grid;
+      grid-gap: var(--outline-slim-gutter);
+      grid-template-columns: 0 1fr auto 0;
```

Comment:
That incredibly sucks

Reply:
Okay, I think i'll move the grid logic into a `@supports` [feature query](https://caniuse.com/?search=supports) as we simply can't do this with regular CSS and it will just have to behave differently on other platforms.

Remind me what webviews we support, again? Where is that documented?

Reply:
On iOS, the Webview is tied to the OS version.

We target iOS 9.0 (IPHONEOS_DEPLOYMENT_TARGET at https://github.com/Jigsaw-Code/outline-client/blob/master/src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj)
and macOS 10.14 (MACOSX_DEPLOYMENT_TARGET at https://github.com/Jigsaw-Code/outline-client/blob/master/src/cordova/apple/xcode/osx/Outline.xcodeproj/project.pbxproj)

It seems iOS 10.3 is the first to support `grid-template-columns`
![image](https://user-images.githubusercontent.com/113565/167504865-5d992fa2-ae2a-46ca-88c7-258dd4956ccb.png)

I don't understand your feature query proposal. Are you going to implement it twice?
I agree with you, this situation isn't great. We could consider stop supporting older devices, but it seems we still have a good number of them.

Perhaps a better alternative is to revert to the html that was already working?

Reply:
Er, let's discuss this offline, I think there's a few things we gotta align on here. Some of the improvements we can't do without grid.

BTW, no need to spend the time getting the screenshot haha, I already mentioned grid started support in iOS 10 (sure, 10.3 I guess)